name: face-recognition
channels:
  - conda-forge
  - defaults
dependencies:
  - ca-certificates=2019.3.9=hecc5488_0
  - certifi=2019.3.9=py37_0
  - flask=1.0.3=py_0
  - flask-cors=3.0.7=py_0
  - intel-openmp=2019.3=203
  - itsdangerous=1.1.0=py_0
  - jinja2=2.10.1=py_0
  - libblas=3.8.0=10_mkl
  - libcblas=3.8.0=10_mkl
  - liblapack=3.8.0=10_mkl
  - markupsafe=1.1.1=py37hfa6e2cd_0
  - mkl=2019.3=203
  - numpy=1.16.3=py37h873a0b8_0
  - openssl=1.1.1b=hfa6e2cd_2
  - pandas=0.24.2=py37h6538335_0
  - pip=19.1.1=py37_0
  - python=3.7.3=hb12ca83_0
  - python-dateutil=2.8.0=py_0
  - pytz=2019.1=py_0
  - setuptools=41.0.1=py37_0
  - six=1.12.0=py37_1000
  - sqlite=3.28.0=hfa6e2cd_0
  - vc=14=0
  - vs2015_runtime=14.0.25420=0
  - werkzeug=0.15.4=py_0
  - wheel=0.33.4=py37_0
  - wincertstore=0.2=py37_1002
  - pip:
    - click==7.0
    - dlib==19.17.0
    - face-recognition==1.2.3
    - face-recognition-models==0.3.0
    - pillow==6.0.0
prefix: C:\Users\andres\Anaconda3\envs\face-recognition
